### PR TITLE
Issue 128

### PR DESCRIPTION
**Pull request by XuechunHou**
#128

### CODE REVIEW

File: web_server/authors/views.py [hunk 1]
```diff
@@ -109,31 +109,31 @@ def update_author_profile(request, author_id):
...
 
     github = body.get('github', None)
     if github is None:
-        return HttpResponse("Post body missing fields: github", status=404)
+        return HttpResponse("Post body missing fields: github", status=422)
 
     display_name = body.get('display_name', None)
     if display_name is None:
-        return HttpResponse("Post body missing fields: display_name", status=404)
+        return HttpResponse("Post body missing fields: display_name", status=422)
```
Comment (AustinGrey) [inline web_server/authors/views.py]: This is a lot of code duplication. Is there no way to simplify this?

**Review by AustinGrey — CHANGES_REQUESTED** [review]

**Comment by XuechunHou** [comment]
changes made. removed duplicate code.

File: web_server/authors/views.py [hunk 2]
```diff
@@ -225,10 +211,11 @@ def create_new_post(request):
...
         # new_post.source      = post['source']       #: "http://lastplaceigotthisfrom.com/posts/yyyyy"
         # new_post.origin      = post['origin']       #: "http://whereitcamefrom.com/posts/zzzzz"
-        new_post.description = post['description']  # : "This post discusses stuff -- brief",
+        # : "This post discusses stuff -- brief",
+        new_post.description = post['description']
         new_post.contentType = post['contentType']  # : "text/plain",
-        new_post.content     = post['content']      #: "stuffs",
-        new_post.author      = request.user         # the authenticated user
+        new_post.content = post['content']      #: "stuffs",
+        new_post.author = request.user         # the authenticated user
```
Comment (AustinGrey) [inline web_server/authors/views.py:218]: This is an example of an area of the code you keep changing that is reducing readability. It's not a huge issue so I'm going to approve and merge, but please prevent your IDE from automatically changing the code others keep formatting to keep readable.

**Review by AustinGrey — APPROVED** [review]
Great solution, much cleaner. Could you please again check that you are not auto

**Pull request merged**